Sources/Fullscreen/FullscreenGallery/FullscreenImageView.swift

```diff
@@ -0,0 +1,167 @@
+//
+// Copyright (c) 2019 FINN AS. All rights reserved.
+//
+
+import UIKit
+
+class FullscreenImageView: UIScrollView {
+
+    // MARK: - Public properties
+
+    public var image: UIImage? {
```

Comment:
I think you can remove `public` from vars and functions in this class since it's internal.

Reply:
Still getting used to the Swift-modifiers... 🙃